portal/src/LoadingModal.module.scss

```diff
@@ -0,0 +1,9 @@
+.root {
```

Comment:
Just FYI, implementing a modal is not that easy. For example, 'position: absolute' may fail when body can scroll. If we want to use modal, we can use the [`Modal` component](https://developer.microsoft.com/en-us/fluentui#/controls/web/modal) provided by Fluent UI, which *should* be more well-tested than our hand-rolled one.